content/assets/stylesheets/_base.scss

```diff
@@ -13,3 +13,33 @@
 .reference-body h3+h4 {
   margin-top: 1em;
 }
+
+.scroll-top {
```

Comment:
Styling must be reusing the maximum from http://styleguide.bookingsync.com/
If something missing it certainly need to be added in the styleguide itself.